code/hud/hudtarget.cpp

```diff
@@ -1360,7 +1362,9 @@ static object* select_next_target_by_distance(const bool targeting_from_closest_
                 continue;
             }
 
-            new_distance = hud_find_target_distance(prospective_victim_ptr, Player_obj);
+			// we need to use the same distance method as evaluate_ship_as_closest_target
```

Comment:
this is very nitpicky, but it looks like there is an extra whitespace tab?

Reply:
It turns out this function indented using spaces whereas I used tabs.  I changed my indentation in this function.